NEWS.md

```diff
@@ -2,6 +2,7 @@
 * [MODSOURMAN-516](https://issues.folio.org/browse/MODSOURMAN-516) Send QM_COMPLETED event after processing finished
 * [MODSOURMAN-517](https://issues.folio.org/browse/MODSOURMAN-517) Change quickMarc producers not to close after message sent
 * [MODSOURMAN-524](https://issues.folio.org/browse/MODSOURMAN-524) Support MARC Holdings
+* [MODDICORE-505](https://issues.folio.org/browse/MODDATAIMP-505)  Update Folijet Rancher
```

Comment:
Please write a message that reflects the changes - decision to  improve logging was made during troubleshooting Rancher env, but it is not related to the code changes
```suggestion
* Improved logging
```